python/test_datago_db.py

```diff
@@ -0,0 +1,186 @@
+from datago import datago
```

Comment:
these are new from this PR, still missing a few from the internal codebase but almost there